src/main/resources/2.7.x.conf

```diff
@@ -0,0 +1,149 @@
+
+          "sbt-less" = """addSbtPlugin("com.typesafe.sbt" % "sbt-less" % "1.1.2")"""
+
+          "sbt-jshint" = """addSbtPlugin("com.typesafe.sbt" % "sbt-jshint" % "1.0.6")"""
+
+          "sbt-rjs" = """addSbtPlugin("com.typesafe.sbt" % "sbt-rjs" % "1.0.10")"""
+
+          "sbt-digest" = """addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")"""
+
+          "sbt-mocha" = """addSbtPlugin("com.typesafe.sbt" % "sbt-mocha" % "1.1.2")"""
+
+          "sbt-sassify" = """addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.12")"""
+
+          "gatling-sbt" = """addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.2")"""
+        }
+      },
+
+      {
+        path = "**build.gradle"
+        conversions = {
+
+          "def scalaVersion" = """def scalaVersion = System.getProperty("scala.binary.version", /* default = */ "2.12")"""
+          "def playVersion" = """def playVersion = "2.7.0-RC3""""
+
+          """"com.typesafe.play:anorm"""" = """    play "com.typesafe.play:anorm_$scalaVersion:2.5.3""""
+
+          """"com.typesafe.play:play-slick"""" = """    play "com.typesafe.play:play-slick_$scalaVersion:4.0.0-RC1""""
+
+          """"com.typesafe.play:play-slick-evolutions"""" = """    play "com.typesafe.play:play-slick-evolutions_$scalaVersion:4.0.0-RC1""""
+
+          """"com.softwaremill.macwire:macros"""" = """    play "com.softwaremill.macwire:macros_$scalaVersion:2.3.1""""
+
+          # play-java-dagger2-example
+          """"com.google.dagger:dagger"""" = """    play "com.google.dagger:dagger:2.16""""
+          """"com.google.dagger:dagger-compiler"""" = """    play "com.google.dagger:dagger-compiler:2.16""""
+
+          "scala-guice" = """    play "net.codingwell:scala-guice_$scalaVersion:4.2.1""""
+          "gatling-charts-highcharts" = """    playTest "io.gatling.highcharts:gatling-charts-highcharts:$gatlingVersion""""
+          "gatling-test-framework" = """    playTest "io.gatling:gatling-test-framework:$gatlingVersion""""
+
+          "com.h2database" = """    play "com.h2database:h2:1.4.197""""
+
+          "scalatestplus-play" = """    playTest "org.scalatestplus.play:scalatestplus-play_$scalaVersion:4.0.0-RC1""""
+        }
+      },
+
+      {
+        path = "**/test-gradle"
+        conversions = {
+          "./gradlew -Dscala.binary.version" = "./gradlew -Dscala.binary.version=$scala_binary_version check -i --stacktrace"
+        }
+      },
+
+      {
+        # Keep the gradle version updated
+        path = "**gradle/wrapper/gradle-wrapper.properties"
+        conversions = {
+          "distributionUrl" = """distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip"""
+        }
+      }, 
+
+      {
+        # Keep the gradle version updated
+        path = ".travis.yml"
+        conversions = {
+          "- oraclejdk10" = """- openjdk10"""
+
+           "jdk: oraclejdk10" = """      jdk: openjdk10"""
```

Comment:
Extra spaces.

Reply:
this are needed, that's for travis yml file

Reply:
Oh, so it doesn't honour the spaces that are in front of "jdk: oraclejdk10"?

Reply:
no, it detects a line containing it an replace the whole line

As I said, TemplateControl deserves some love. Especially, if we move into more automation and want to make release cheaper.

Reply:
Yeah, sounds like it!